python/kfserving/README.md

```diff
@@ -6,14 +6,16 @@ KFServing provides many funtionalities, including among others:
 
 * Registering a model and starting the server
 * Prediction Handler
-* Liveness Handler 
-* Metrics Handler 
+* Liveness Handler
+* Metrics Handler
 
 KFServing supports the following storage providers:
 
 * Google Cloud Storage with a prefix: "gs://"
 * S3 Compatible Object Storage with a prefix "s3://"
-* Local filesystem with a prefix "/"
+* Local filesystem with a prefix "file://". For example:
+    * With absolute path: `file:///absolute/path`
+    * With relative path: `file://./relative/path` or `file://relative/path`
```

Comment:
`file://./` feels really awkward.

Reply:
So I added back the `os.path.exists` case now, users can either use the `file://` prefix or without any prefix.